README.md

```diff
@@ -65,4 +65,7 @@ As stated in <cite>[Ansible-Elasticsearch][1]</cite>.
 > es_use_repository is true) if you want to use a repo already present"
 
 
+`es_add_repository` set via `group_vars/all.yml` will also influence the existence of the Elasticsearch/Kibana repository on Kibana hosts. The default valulue inside the `kibana` role is `true`.
```

Comment:
```suggestion
`es_add_repository` set via `group_vars/all.yml` will influence the existence of the Elasticsearch/Kibana repository on Kibana hosts. The default value inside the `kibana` role is `true`.
```

Reply:
`Also` was included since two sentences or so above we already talk about the variable `es_add_repository` and in what way it is relevant for the elasticsearch role

Reply:
So `es_add_repository` can be influenced via two different places?

Reply:
setting wise you could actually set the variable in a lot more places than two - but that is default for Ansible.
The main point was only to mention here that the variable is used in two roles.